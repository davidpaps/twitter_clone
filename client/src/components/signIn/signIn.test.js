import React from "react";
import { shallow } from "enzyme";
import SignIn from "./signIn";

describe("Sign In", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<SignIn />)));

  it("should render a <Fragment />", () => {
    expect(wrapper.find("Fragment").length).toEqual(1);
  });

  it("should render a <form />", () => {
    expect(wrapper.find("form").length).toEqual(1);
  });

  it("should render a <Input/>", () => {
    expect(wrapper.find("input").length).toEqual(2);
  });

  it("should render a <button/>", () => {
    expect(wrapper.find("button").length).toEqual(2);
  });

  it("should render a <Link />", () => {
    expect(wrapper.find("Link").length).toEqual(1);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
