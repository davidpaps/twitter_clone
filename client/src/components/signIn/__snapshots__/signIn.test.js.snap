// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign In should render correctly 1`] = `ShallowWrapper {}`;
